creds.json

```diff
@@ -1,5 +1,13 @@
 {
-  "admin": "admin",
-  "cvasquez": "cristian",
-  "tpluscode": "tomasz"
+  "admin": {
+    "name": "The View Builder",
+    "password": "admin"
+  },
+  "cvasquez": {
+    "name": "Cristian Vasquez",
+    "password": "cristian"
+  },
+  "tpluscode": {
+    "password": "tomasz"
+  }
```

Comment:
@ludovicm67 here's how I changed the JSON. note that the `name` is optional. Will use the key for display when missing

Reply:
OK, thanks for the info.
I updated the credentials file that is used, so that the PR is ready to merge ;)